### DISCUSSION TITLE
Revision to Manual: Getting started with MTEX.  Any requests

### DISCUSSION THREAD

**Discussion by jhiscocks** [mtex-toolbox/mtex#527] [Ask Anything]
Hello All, I'm doing a major rewrite of my documentation [Getting started with MTEX for EBSD analysis](https://www.researchgate.net/profile/Jessica_Hiscocks/publication/314280728_Getting_started_with_MTEX_for_EBSD_analysis/links/58befd29aca272bd2a3ace95/Getting-started-with-MTEX-for-EBSD-analysis).  I'll be adding a lot of images and clarifying explanations.  Anything you'd like to have added in terms of content?  What were the most frustrating tasks when you were first using MTEX?

Cheers, Jessica.

**Comment by ralfHielscher** [comment]
Hi Jessica,

thank you a lot for all your efforts in this direction. I'm also curious about the answers. I guess you will get more answers if you ask this question additionally on researgate.

Ralf.

**Reply by ralfHielscher** [reply]
I also edited your post to include a link to your documentation. I think this makes your question more clear.

Ralf.

**Comment by nyyssont** [comment]
I'd say any tips you can put in on how to relate MTEX orientations to those of other systems will be worth their weight in gold! I know there's a good section about that in the documentation but I think this information cannot be repeated enough.

**Comment by nyyssont** [comment]
I just discovered that the online documentation is written as m-files that can be run straight up in Matlab. I find this quite useful and I think this should be mentioned!
